apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -490,6 +516,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         if (!_canRepresentativesVote(vote_)) {
             return false;
         }
+        // Note: should we also check that the length is <= MAX_VOTES_DELEGATION_SET_LENGTH? Perhaps better to keep this as a revert, similar to a missing vote
```

Comment:
Thought this might be useful to mirror `voteOnBehalfOf()`'s behaviour.

Reply:
Addressed in f25c3ca02101a272f9bd6d698951f78da20c6f9c